web/src/Web.App/appsettings.Development.json

```diff
@@ -21,18 +21,6 @@
     "ContainerName": "sessions",
     "DatabaseName": "session-data"
   },
-  "DFESignInSettings": {
```

Comment:
Personally I like keeping these here, even though they get overridden by dotnet user secrets. It lets the Devs know that there is some additional configuration required, and the base values can be copy-and-pasted directly into the relevant `secrets.json` file if required, rather than using the CLI.

Reply:
I have added this to the README.md now. My worry is keeping those settings and people updating the settings and not realising that it is actually the user-secrets they need to update, when they have both set, it is easy to forget. At least if they don't have these settings in the first place it would just fail.

While I don't have a strong opinion if this is the approach we are going to take having the secrets templated both in here and in the secrets. We should then make sure we do it for all settings.

Reply:
@colinbull I'll pick up updating the readme with templated secrets files on a separate PR.